quality-assurance/safety-assurance-guidance.md

```diff
@@ -1,44 +1,47 @@
-# Safety Assurance Guidelines
+This policy describes the Software Engineering Team’s policy and process for ensuring our SOLAS products are created and changed with appropriate consideration given to safety.
 
-## What is Safety Modelling
+# Goals
 
-Safety Modelling is focused around two key areas:
+-	Minimise the causes of product delay and inaccuracy introduced through the software we create;
+-	Provide traceability for the delivery of safety requirements;
+-	Create full delivery team awareness of the safety impact of the software they create;
+-	Prompt high-value team conversations about safety;
+-	Not burden teams with low-value bureaucracy;
+-	Provide safety-related analysis on a just-in-time basis, rather than through wasteful up-front effort.
 
-* Error (Data Integrity)
-* Delay
+# Overview of approach
 
-This is centered around asking *What if* questions to determine safety risks and the likelihood and safety impact of those risks occurring.
+Our approach is closely related to our successful Developer Threat Modelling process which is used as part of our security lifecycle.
```

Comment:
Can this be linked somewhere - maybe not.

Reply:
Not currently, nothing in GitHub.  I will keep an eye out for an opportunity